test/e2e/buildx_inspect_test.go

```diff
@@ -0,0 +1,209 @@
+//go:build linux || freebsd
+
+package integration
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/containers/podman/v5/cmd/podman/registry"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman buildx inspect", func() {
+	It("podman-remote buildx inspect", func() {
+		if !registry.IsRemote() {
+			Skip("This test is only applicable for remote podman")
+		}
```

Comment:
Why are these skipped on remote? Looking at the podman code the command has a branch for if remote but it should work fine. All tests here skip on remote which means the IsRemote() condition in the actual comman is never executed.

Second calling `registry.IsRemote()` is actually incorrect from test context and doesn't do what you want. If you really have to skip on remote one must use `SkipIfRemote()` which uses `IsRemote` from the e2e package. Notice how this is using the remote_testing build tag to switch the behaviour. registry.IsRemote() passes cli arguments which makes no sense in the context of ginkgo tests.